prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -72,19 +103,6 @@ class BlobAccess(
         return stream.toByteArray()
     }
 
-    /**
```

Comment:
Copying is used for the HHSProtect.
I was leaving this code there, because its a better way to copy, but time for it to go, I guess.  So you know, I wasn't able to get this working, so abandoned it, and switched to the copyBlob below that pulls data into memory, then writes out again.   Less efficient, but it works.    We have an open ticket (I think) to fix this.